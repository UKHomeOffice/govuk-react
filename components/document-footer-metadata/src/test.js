import React from 'react';
import ReactDOM from 'react-dom';
import DocumentFooterMetadata from './';

describe(DocumentFooterMetadata, () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<DocumentFooterMetadata>Example</DocumentFooterMetadata>, div);
  });
});
