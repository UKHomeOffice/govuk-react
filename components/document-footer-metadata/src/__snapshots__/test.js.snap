// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styled matches snapshot: enzyme.mount 1`] = `
.emotion-0 {
  font-family: "nta",Arial,sans-serif;
}

.emotion-3 {
  margin: 0;
  padding: 0;
  padding-left: 20px;
  font-family: "nta",Arial,sans-serif;
  font-weight: 400;
  text-transform: none;
  font-size: 14px;
  line-height: 1.1428571429;
  list-style-type: none;
  margin-left: 0;
  padding-left: 0;
}

@media only screen and (min-width:641px) {
  .emotion-3 {
    font-size: 16px;
    line-height: 1.25;
  }
}

.emotion-1 {
  font-size: 24px;
  font-weight: 700;
  line-height: 1.25;
}

.emotion-1 > a {
  -webkit-text-decoration: none;
  text-decoration: none;
}

<Styled(DocumentFooterMetadata)
  partOf={
    Array [
      "example",
      "exampleexample",
    ]
  }
>
  <DocumentFooterMetadata
    className="emotion-4"
    partOf={
      Array [
        "example",
        "exampleexample",
      ]
    }
  >
    <div>
      <Styled(div)>
        <div
          className="emotion-0"
        />
      </Styled(div)>
      <Styled(div)>
        <div
          className="emotion-0"
        >
          <div>
            <p
              style={
                Object {
                  "marginBottom": 0,
                }
              }
            >
              Part of:
            </p>
            <Styled(UnorderedList)
              listStyleType="none"
            >
              <UnorderedList
                className="emotion-4"
                listStyleType="none"
              >
                <Styled(ul)
                  className="emotion-4"
                  listStyleType="none"
                >
                  <ul
                    className="emotion-3"
                  >
                    <Styled(li)
                      key="0"
                    >
                      <li
                        className="emotion-1"
                      >
                        example
                      </li>
                    </Styled(li)>
                    <Styled(li)
                      key="1"
                    >
                      <li
                        className="emotion-1"
                      >
                        exampleexample
                      </li>
                    </Styled(li)>
                  </ul>
                </Styled(ul)>
              </UnorderedList>
            </Styled(UnorderedList)>
          </div>
        </div>
      </Styled(div)>
      <Styled(div)>
        <div
          className="emotion-0"
        />
      </Styled(div)>
    </div>
  </DocumentFooterMetadata>
</Styled(DocumentFooterMetadata)>
`;
